host/i3_workspaces.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "i3_workspaces",
   "description": "Communicate with i3 about which workspaces Firefox windows are on",
-  "path": "/home/yuri/projects/firefox-i3-workspaces/host/i3_workspaces.py",
+  "path": "/usr/bin/i3_workspaces.py",
```

Comment:
Please keep that as your ebuild’s private patch. The value in the upstream manifest is supposed to serve as an example for users installing manually, and those should prefer to install in their home, which might be the only part of the filesystem that is writable by them.

Also, note that `/usr/bin` is for user-facing executables, i.e. programs users will typically want to run from the shell prompt. A backend for a browser extension is not such a program. It is started by the browser behind the scenes. So it belongs in `/usr/lib` or `/usr/libexec`. (See Filesystem Hierarchy Standard [§ 4.6](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#usrlibLibrariesForProgrammingAndPa) and 4.7, and check if your GNU/Linux distribution has any specific policy in favor of one or the other.)